tests/test_m2m.py

```diff
@@ -0,0 +1,159 @@
+import unittest
+from m2m import m2m
+
+class MetadataRecordTests(unittest.TestCase):
+
+    def testMetadataRecordSetup(self):
+        # test that the creation of a record creates a MetadataRecord object.
+
+        self.record = m2m.MetadataRecord("mphillips")
```

Comment:
do all these really need to be prefixed with self? You're recreating the same var in each test, so it doesn't seem like there's any point to making the var accessible between methods.